backend/dataall/aws/handlers/sts.py

```diff
@@ -168,6 +168,17 @@ def get_cdk_look_up_role_arn(cls, accountid, region):
         log.info(f"Getting CDK look up role: arn:aws:iam::{accountid}:role/cdk-hnb659fds-lookup-role-{accountid}-{region}")
         return 'arn:aws:iam::{}:role/cdk-hnb659fds-lookup-role-{}-{}'.format(accountid, accountid, region)
 
+    @classmethod
+    def get_cdk_exec_role_arn(cls, accountid, region):
+        """Returns the name that will be assumed to perform IAM actions on a given AWS accountid using CDK Toolkit role
+        Args:
+            accountid(string) : aws account id
+        Returns:
+                string : arn of the CDKToolkit role on the target aws account id
+        """
+        log.info(f"Getting CDK exec role: arn:aws:iam::{accountid}:role/cdk-hnb659fds-lookup-role-{accountid}-{region}")
```

Comment:
Very minor - change the log.info to make sure we know that we get the exec role

Reply:
good catch